storage/testdb/testdb.go

```diff
@@ -33,7 +33,7 @@ import (
 
 var (
 	trillianSQL = testonly.RelativeToPackage("../mysql/storage.sql")
-	dataSource  = "root@/"
+	dataSource  = "root@tcp(127.0.0.1)/"
```

Comment:
According to https://github.com/go-sql-driver/mysql#dsn-data-source-name, TCP to "localhost" is the default. How is 127.0.0.1 different to the default?